apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/RdaServer.java

```diff
@@ -0,0 +1,45 @@
+package gov.cms.bfd.pipeline.rda.grpc.server;
+
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.grpc.inprocess.InProcessServerBuilder;
+import java.io.IOException;
+import java.util.function.Supplier;
+
+public class RdaServer {
+  /**
+   * Creates a local RDA API server on a random port for testing.
```

Comment:
why a random port, wouldn't we want to control this for our tests?

Reply:
There is an overload of this method farher down that accepts a specific port to use.  The random port option is intended for IT's rather than any sort of long term deployment of the server on an test instance.